src/Redis/Tests/Caching/RedisCacheClientTests.cs

```diff
@@ -68,8 +69,19 @@ public class RedisCacheClientTests : CacheClientTestsBase {
         }
 
         [Fact]
-        public override Task CanManageSets() {
-            return base.CanManageSets();
+        public override async Task CanManageSets() {
+            await base.CanManageSets();
+
+            var cache = GetCacheClient();
+            if (cache == null)
+                return;
+
+            using (cache) {
+                await Assert.ThrowsAsync<StackExchange.Redis.RedisServerException>(async () => {
```

Comment:
Yeah, we talked about that, but the problem is that redis throws generic exceptions so we can't really handle that error specifically.